client/my-sites/stats/activity-log/index.jsx

```diff
@@ -609,22 +625,9 @@ class ActivityLog extends Component {
 				{ this.renderErrorMessage() }
 				{ hasFirstBackup && this.renderMonthNavigation() }
 				{ this.renderActionProgress() }
-				{ ! requestData.logs.hasLoaded && (
-					<section className="activity-log__wrapper">
-						<ActivityLogDayPlaceholder />
-						<ActivityLogDayPlaceholder />
-						<ActivityLogDayPlaceholder />
-					</section>
-				) }
```

Comment:
why not just add a check for `isEmpty( logs )` here instead of extracting this into a separate variable?

Reply:
I thought nested ternaries were forbidden. I made the change.

Reply:
uh…who said anything about nested ternaries?

Reply:
Yes, nested ternaries are forbidden

/home/ubuntu/wp-calypso/client/my-sites/stats/activity-log/index.jsx
  612:7  error  Do not nest ternary expressions  no-nested-ternary

✖ 1 problem (1 error, 0 warnings)

Reply:
I wasn't suggesting nested ternaries. it appeared like the logic in this PR could have been recreated by only adding one or two more checks in the conditions already in the code. no ternaries required.

Reply:
I won't put another `isEmpty` because I want to test it only once.

Reply:
don't worry about it. the test is cheap.